README.md

```diff
@@ -94,6 +94,51 @@ module.exports = {
 };
 ```
 
+### Additional Scenarios
+Scenario 1
+> Alice is using a CSS Extractor and her images are not loading successfully. In the loader, she should apply the ```encoding``` parameter with a value of ```'base64'``` to ensure quotes are added to her url(...) call.
```

Comment:
Please clarify what it `CSS Extractor`.

`encoding: 'base64'` will make loader to encode images with using base64 encoding, just as url-loader does.  
Quotes should not be added even without base64 option. I mean you should not have issues because of additional quotes.